foreman-bootloaders-redhat/foreman-bootloaders-redhat.spec

```diff
@@ -1,108 +1,74 @@
-%define _binaries_in_noarch_packages_terminate_build 0
-
 Name: foreman-bootloaders-redhat
-Version: 201707171807
+Version: 201801241201
 Release: 1%{?dist}
-Summary: Binaries of Grub1, Grub2 and Shim bootloaders
+Summary: Metapackage with Grub2 and Shim TFTP bootloaders
 
 Group: Applications/System
 License: GPLv2+ and GPLv3+ and BSD
-URL: https://github.com/theforeman/foreman-bootloaders
-Source0: http://downloads.theforeman.org/foreman-bootloaders/%{name}-%{version}.tar.bz2
+URL: https://github.com/theforeman/foreman
 BuildArch: noarch
 
+Source0: foreman-generate-bootloaders
+
+Requires: grub2-tools-extra
+Requires: grub2-efi-ia32-modules
+Requires: grub2-efi-x64-modules
+Requires: grub2-pc-modules
+Requires: shim-ia32
+Requires: shim-x64
 
 %description
-The package contains binaries of Grub and Grub2 bootloaders for PXE/BOOTP use
-on TFTP Foreman Proxy extracted from Red Hat Enterprise Linux 6 and 7. This
-noarch package contains binaries for multiple architectures, including x86_64,
-i386, ARM64, PPC64 and PPC64 LE.
+Package containing foreman-generate-bootloaders helper script which generates and copies
+Grub2 and Shim bootloaders for PC and EFI into /var/lib/tftpboot.
 
 %package tftpboot
 Summary: Copies of bootloaders for PXE
 Group: Applications/System
 Requires: coreutils
 Requires: %{name}
 Requires: /var/lib/tftpboot
-Requires: foreman-proxy
```

Comment:
if that is required for /var/lib/tftpboot, this should remain here

Reply:
It was required for some subdirectories to be present I think. I will keep it just to prevent any possible regression. Thanks.